components/vr_traits/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "vr_traits"
+version = "0.0.1"
+authors = ["The Servo Project Developers"]
+license = "MPL-2.0"
+publish = false
+
+[lib]
+name = "vr_traits"
+path = "lib.rs"
+
+[dependencies]
+ipc-channel = "0.5"
+msg = {path = "../msg"}
+serde = "0.8"
+serde_derive = "0.8"
+rust-webvr = {version = "0.1", features = ["serde-serialization"] }
+
+
```

Comment:
feel free to remove these two superfluous newlines, but keep a single one at EOF